tests/polyfill/test_h3.py

```diff
@@ -400,7 +398,7 @@ def test_multipoly_checks():
 def test_3d_geo():
     loop = lnglat_open()
 
-    loop2d = [(lng, lat)      for lng, lat in loop]
+    loop2d = [(lng, lat) for lng, lat in loop]
```

Comment:
Worse

Reply:
Why? Is there any benefit of having the `for` in the two loops aligned?